### PR TITLE
feat(REST): Added parameter to GET project by Group and Tag

### PR DESCRIPTION
**Pull request by JaideepPalit**
Added parameter to GET project by Group and Tag

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

### CODE REVIEW

**Comment by smrutis1** [comment]
@JaideepPalit , Kindly consider adding documentation for these two new parameter.

**Comment by JaideepPalit** [comment]
Added Documentation for the newly added parameters.

File: rest/resource-server/src/docs/asciidoc/projects.adoc [hunk 1]
```diff
@@ -69,6 +69,42 @@ include::{snippets}/should_document_get_projects_by_type/http-response.adoc[]
...
+include::{snippets}/should_document_get_projects_by_group/curl-request.adoc[]
+
+===== Example response
+include::{snippets}/should_document_get_projects_by_group/http-response.adoc[]
+
+===== Links
+include::{snippets}/should_document_get_projects_by_group/links.adoc[]
+
+
+[[resources-projects-list-by-group]]
```
[the remark this answers is not in the document]
Comment (JaideepPalit) [reply rest/resource-server/src/docs/asciidoc/projects.adoc]: Changes made as suggested.

File: rest/resource-server/src/docs/asciidoc/projects.adoc [hunk 2]
```diff
@@ -69,6 +69,42 @@ include::{snippets}/should_document_get_projects_by_type/http-response.adoc[]
 include::{snippets}/should_document_get_projects_by_type/links.adoc[]
 
 
+[[resources-projects-list-by-group]]
+==== Listing by group
+
+A `GET` request will list all of the service's projects by group.
```
[the remark this answers is not in the document]
Comment (JaideepPalit) [reply rest/resource-server/src/docs/asciidoc/projects.adoc]: Changes made as suggested.

File: rest/resource-server/src/docs/asciidoc/projects.adoc [hunk 3]
```diff
@@ -69,6 +69,42 @@ include::{snippets}/should_document_get_projects_by_type/http-response.adoc[]
...
+include::{snippets}/should_document_get_projects_by_group/http-response.adoc[]
+
+===== Links
+include::{snippets}/should_document_get_projects_by_group/links.adoc[]
+
+
+[[resources-projects-list-by-group]]
+==== Listing by tag
+
+A `GET` request will list all of the service's projects by tag.
```
[the remark this answers is not in the document]
Comment (JaideepPalit) [reply rest/resource-server/src/docs/asciidoc/projects.adoc]: Changes made as suggested.

**Pull request merged**